### PR TITLE
Extra fixes to make SF6 PR pass checks

### PR DESCRIPTION
**Pull request by acrobat**

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -65,15 +65,7 @@ jobs:
...
-                  tools: 'composer:v2'
-
-            - name: Globally install symfony/flex
-              if: matrix.symfony-require != ''
-              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
-
-            - name: Remove dev dependencies not compatible with Symfony 6
-              if: matrix.symfony-require == '6.0.*'
-              run: composer remove liip/rmt --dev --no-update --no-interaction
+                  tools: 'composer:v2, flex'
```
Comment (dbu) [inline .github/workflows/test-application.yaml:68]: should we have the `,` or not? here you have one, below you don't

**Review by dbu — CHANGES_REQUESTED** [review]
thanks a lot for the cleanups and for finding the problem with that assertion!

can you please check whether the tools value should be comma separated or just spaces?

File: .github/workflows/test-application.yaml (hunk 1 above)
Comment (acrobat) [reply .github/workflows/test-application.yaml:68]: Good catch, was copy paste error. It should indeed be seperated by a comma! Fixed!

Comment (dbu) [reply .github/workflows/test-application.yaml:68]: yep, previously it only installed flex for dbal

**Review by dbu — APPROVED** [review]

**Pull request merged**